scripts/event_collectors/service_status.sh

```diff
@@ -20,17 +20,69 @@
 # Constants
 # ----------------------------------------------------------------------------
 STATE_TYPE='service'
-SERVICES='firemasq'
 
+check_each_system_service() {
+  service_name=$1
+  state_expected=$2
+  state_actual=$(sudo systemctl is-active $service_name)
+  test  "$state_actual" == "$state_expected"; _rc1=$?
+  echo "state $STATE_TYPE $service_name $_rc1 state_actual=$state_actual state_expected=$state_expected"
+  return $_rc1
+}
+
+check_services() {
+    _rc=0
+    check_each_system_service fireapi "active" || _rc=1
+    check_each_system_service firemain "active" || _rc=1
+    check_each_system_service firemon "active" || _rc=1
+    check_each_system_service firekick "inactive" || _rc=1
+    check_each_system_service redis-server "active" || _rc=1
+    check_each_system_service brofish "active" || _rc=1
+    check_each_system_service firewalla "inactive" || _rc=1
+    check_each_system_service fireupgrade "inactive" || _rc=1
+    check_each_system_service fireboot "inactive" || _rc=1
+
+    vpn_state=$(redis-cli hget policy:system vpn | jq .state)
+    $vpn_state && vpn_run_state='active' || vpn_run_state='inactive'
+    check_each_system_service openvpn@server $vpn_run_state || _rc=1
+
+    if [[ $PLATFORM != 'gold' ]]; then # non gold
```

Comment:
use env MANAGED_BY_FIREROUTER, do not hard code gold here

Reply:
fixed as suggested